### PR TITLE
second attempt for first successful PR

### PR DESCRIPTION
**Pull request by asenpetrov**

### CODE REVIEW

File: Exercise01-Git/Asen-Petrov/helloWords-Asen-Petrov.js
Comment (karastoyanov) [inline Exercise01-Git/Asen-Petrov/helloWords-Asen-Petrov.js:1]: Change the name of the file to **helloWorld**-Asen-Petrov.js

File: Exercise02-Basics/Asen-Petrov/basics01-004.js
Comment (karastoyanov) [inline Exercise02-Basics/Asen-Petrov/basics01-004.js:1]: Try to use the knowedge from Lesson03 and solve the exercise without arrays and loops.

File: Exercise01-Git/Asen-Petrov/helloWords-Asen-Petrov.js
Comment (asenpetrov) [reply Exercise01-Git/Asen-Petrov/helloWords-Asen-Petrov.js:1]: file name changed

File: Exercise02-Basics/Asen-Petrov/basics01-004.js
Comment (asenpetrov) [reply Exercise02-Basics/Asen-Petrov/basics01-004.js:1]: Loops and arrays removed, variable set and printed separately.

**Review by karastoyanov — COMMENTED** [review]
Current progress has been approved.

**Comment by asenpetrov** [comment]
Any pending actions on my end before being able to merge the PR?

File: Exercise02-Basics/Asen-Petrov/basics02-013.js
Comment (karastoyanov) [inline Exercise02-Basics/Asen-Petrov/basics02-013.js:1]: You can try and solve the task with ternary operators. The solution would be something like that:

```javascript
let result = (letter >= 'A' && letter <= 'Z') ? CHANGE THE UPPERCASE TO LOWERCASE EXPRESSION : (letter >= 'a' && letter <= 'z') ? CHANGE TO LOWERCASE TO UPPERCASE EXPRESSION : letter;
```

File: Exercise02-Basics/Asen-Petrov/basics02-014.js
Comment (karastoyanov) [inline Exercise02-Basics/Asen-Petrov/basics02-014.js:1]: You can try and solve the task with ternary operators. The solution would be something like that:

```javascript
let result = (letter >= 'A' && letter <= 'Z') ? CHANGE THE UPPERCASE TO LOWERCASE EXPRESSION : (letter >= 'a' && letter <= 'z') ? CHANGE TO LOWERCASE TO UPPERCASE EXPRESSION : letter;
```

File: Exercise02-Basics/Asen-Petrov/basics02-015.js
Comment (karastoyanov) [inline Exercise02-Basics/Asen-Petrov/basics02-015.js:1]: The code is fine, but you can try to solve it without if/else statements.

* First, we calculate the index of the ciphered letter by converting the given letter to its ASCII code, subtracting 65 to map 'A' to 0, adding the Cipher value, and then taking the modulo 26 to ensure it wraps around the alphabet.

* Then, we convert the calculated index back to a letter using String.fromCharCode and print it.

**Review by karastoyanov — CHANGES_REQUESTED** [review]
Take a look on the `BASIC02-014` and `BASIC02-015` tasks, the current solutions may be correct, but we can try to use the knowedge from the initial lessons.

**Review by karastoyanov — APPROVED** [review]

**Comment by karastoyanov** [comment]
BASICS02-014 and BASICS02-015 fixed and commited in order to close the pill-request.

**Pull request merged**